pkg/distro/fedora/package_sets.go

```diff
@@ -188,6 +188,101 @@ func iotCommitPackageSet(t *imageType) rpmmd.PackageSet {
 	return ps
 }
 
+func bootcContainerPackageSet(t *imageType) rpmmd.PackageSet {
+	// Replicating package selection from centos-bootc:
+	// https://github.com/CentOS/centos-bootc/
+	ps := rpmmd.PackageSet{
+		Include: []string{
+			"acl",
+			"attr", // used by admins interactively
+			"bootc",
+			"bootupd",
+			"chrony", // NTP support
+			"container-selinux",
+			"container-selinux",
+			"crun",
+			"cryptsetup",
+			"dnf",
+			"e2fsprogs",
+			"fwupd",                 // if you're using linux-firmware, you probably also want fwupd
+			"iproute", "iproute-tc", // route manipulation and QoS
+			"iptables", "nftables", // firewall manipulation
+			"iptables-services", // additional firewall support
+			"kbd",               // i18n
+			"keyutils",          // Manipulating the kernel keyring; used by bootc
+			"libsss_sudo",       // allow communication between sudo and SSSD for caching sudo rules by SSSD
+			"linux-firmware",    // linux-firmware now a recommends so let's explicitly include it
+			"logrotate",         // There are things that write outside of the journal still (such as the classic wtmp, etc.). auditd also writes outside the journal but it has its own log rotation.  Anything package layered will also tend to expect files dropped in /etc/logrotate.d to work. Really, this is a legacy thing, but if we don't have it then people's disks will slowly fill up with logs.
+			"lsof",
+			"lvm2",                       // Storage configuration/management
+			"nano",                       // default editor
+			"ncurses",                    // provides terminal tools like clear, reset, tput, and tset
+			"NetworkManager-cloud-setup", // support for cloud quirks and dynamic config in real rootfs: https://github.com/coreos/fedora-coreos-tracker/issues/320
+			"NetworkManager", "hostname", // standard tools for configuring network/hostname
+			"NetworkManager-team", "teamd", // teaming https://github.com/coreos/fedora-coreos-config/pull/289 and http://bugzilla.redhat.com/1758162
+			"NetworkManager-tui",               // interactive Networking configuration during coreos-install
+			"nfs-utils-coreos", "iptables-nft", // minimal NFS client
+			"nss-altfiles",
+			"openssl",
+			"ostree",
+			"passwd", "shadow-utils", // User configuration
+			"podman",
+			"rpm-ostree",
+			"selinux-policy-targeted",
+			"sg3_utils",
+			"skopeo",
+			"socat", "net-tools", "bind-utils", // interactive network tools for admins
+			"sssd-client", "sssd-ad", "sssd-ipa", "sssd-krb5", "sssd-ldap", // SSSD backends
+			"stalld",               // Boost starving threads https://github.com/coreos/fedora-coreos-tracker/issues/753
+			"subscription-manager", // To ensure we can enable client certs to access RHEL content
+			"systemd",
+			"systemd-resolved",  // resolved was broken out to its own package in rawhide/f35
+			"tpm2-tools",        // needed for tpm2 bound luks
+			"WALinuxAgent-udev", // ???
```

Comment:
Answering `???` goes like this:

```
git log -G WALinux                                                                                                                                                                                      01/15/24 13:23:50 PM
commit 60d8e77ee405b59ff96ebf7a3a4fd1f9a784272b
Author:     Colin Walters <walters@verbum.org>
AuthorDate: Fri Dec 9 15:30:03 2022 -0500
Commit:     Colin Walters <walters@verbum.org>
CommitDate: Fri Dec 9 15:32:25 2022 -0500

    Initial split from Fedora CoreOS
```

Then running the same process there turns up
```
commit b30c72bc80135c5ff2524272bf5e3d37dd57286c
Author:     Micah Abbott <miabbott@redhat.com>
AuthorDate: Wed Dec 16 13:51:32 2020 -0500
Commit:     Jonathan Lebon <jonathan@jlebon.com>
CommitDate: Wed Jan 6 09:12:08 2021 -0500

    use WALinuxAgent-udev package for Azure udev rules
    
    BZ#1748432 was filed while working on #159 and the udev rules were
    split out into a separate package as part of F32.  Let's stop carrying
    the rules in the overlay and use the provided package now.
```

Now, I was already not totally happy with effectively losing the git history when creating the initial centos-bootc repository, but...at the same time, the rationale is that we're building also isn't the same thing as Fedora CoreOS.

But anyways, I think a notable point here is that we should have a goal to align and keep these things in sync more.

Reply:
Thanks! I updated the comment with a reference to the rhbz.

> I think a notable point here is that we should have a goal to align and keep these things in sync more.

I agree.